server/cron/calendar_cron.go

```diff
@@ -640,7 +640,10 @@ func getCalendarEventDescriptionAndResolution(
 			description = getDefaultDescription()
 			resolution = defaultResolution
 		} else {
-			description = policyDescription
+			if strings.HasSuffix(policyDescription, ".") {
+				policyDescription = policyDescription[:len(policyDescription)-1]
+			}
+			description = fmt.Sprintf("%s (%s).", policyDescription, host.HostDisplayName)
```

Comment:
https://github.com/fleetdm/fleet/pull/20235/files#r1668838526

Same here (I might be confused though.)

Reply:
Thanks for catching these @lucasmrod . The confusion stemmed from the fact that what Google Calendar calls the "Description" actually contains what Fleet is calling a "description" along with a "resolution." I've hopefully clarified all that now.